### ISSUE TITLE
[Bug] Unexpected workspace appears after creating a new workspace

### ISSUE THREAD

**Issue by mustela** [datumforge/datum-ui#204]
After creating a workspace and being redirected to the home page, clicking on the workspaces dropdown shows an additional workspace named "Crack Piglet" which I did not create.

**Steps to reproduce**:

1. Create a new workspace.
2. After being redirected to the home page, click on the workspaces dropdown.
3. Observe that there are two workspaces listed, one of which is named "Crack Piglet" and was not created by the user.

**Expected behavior**:

Only the workspace that was created by the user should appear in the dropdown.

**Comment by golanglemonade** [comment]
@mustela this is the personal workspace, that is created by default. We will hide this unless its the only workspace available.

**Comment by mustela** [comment]
@golanglemonade I don't think you need to hide it entirely; that could be another point confusion. My confusion was because there was no explanation about the personal organization. However, if you change the flow and present it to the user like: 'Hey, thanks for signing up! We've created a personal space so you can test things out. It's a limited environment, etc. Would you like to rename it or create a new workspace?'—something along these lines would be just fine.

**Comment by golanglemonade** [comment]
@mustela Nah; this is actually the flow we want. We want the personal organization to be hidden but there to make our permissions + org transfer functionality easy. Users should be creating a new org to test things out as some of the functionality is not allowed in a personal org (e.g. you cannot invite other members to your personal org, it can't be deleted, etc).
